### PR TITLE
[#908][#906][#905] Legion cli improvements

### LINKED ISSUE #905: Legionctl CLI does not provide ability to get name of trained image
As a user of legionctl, I want to get name of trained image.

### LINKED ISSUE #906: Rename show_metrics
Rename toolchain's API `show_metrics` to `show_local_metrics`, add human-readable description for cloud mode (e.g. `local metrics are unavailable on a cloud mode, please use Grafana instead`)

### LINKED ISSUE #908: Creating MD/MT/VCS from yaml\json files
As a user of legionctl, I want to create a legion entity from a yaml\json file

### PR DESCRIPTION
**Pull request by mcdoker18**
* All legion entities (VCS, MD and MT) can be created, editied or
deleted from files. Supported both k8s and legion format REST API
* Added `show_metrics` function to `show_local_metrics`
* Added warning when `show_local_metrics` function is used on a cloud
* Provide ability to get name of trained image using legionctl

This closes #908 
This closes #906 
This closes #905

### CODE REVIEW

**Comment by mcdoker18** [comment]
bot, test app

File: legion/toolchains/python/legion/toolchain/metrics.py [hunk 1]
```diff
@@ -219,14 +219,18 @@ def clear_metric_store(model_id: str, model_version: str):
...
+def show_local_metrics(model_id: str, model_version: typing.Optional[str] = None) -> pd.DataFrame:
     """
     Show metrics from local store
 
     :param model_id: model ID
     :param model_version: model version
     :return: Metrics which converted to Dataframe
     """
+    if config.MODEL_CLUSTER_TRAIN_METRICS_ENABLED:
+        LOGGER.warning('Local metrics are unavailable on a cloud mode. Use the Grafana dashboard.')
```
Comment (kirillmakhonin) [inline legion/toolchains/python/legion/toolchain/metrics.py]: **in** a cloud mode?

File: legion/sdk/legion/sdk/clients/deployment.py [hunk 2]
```diff
@@ -56,11 +56,11 @@ def from_json(md: typing.Dict[str, str]) -> 'ModelDeployment':
...
             image=md_spec.get('image', ''),
-            resources=md_spec.get('resources', ''),
-            annotations=md_spec.get('annotations', []),
+            resources=md_spec.get('resources', {}),
+            annotations=md_spec.get('annotations', {}),
             replicas=md_spec.get('replicas', ''),
-            liveness_probe_initial_delay=md_spec.get('livenessProbeInitialDelay', ''),
-            readiness_probeInitial_delay=md_spec.get('readinessProbeInitialDelay', ''),
+            liveness_probe_initial_delay=md_spec.get('livenessProbeInitialDelay'),
+            readiness_probeInitial_delay=md_spec.get('readinessProbeInitialDelay'),
```
Comment (kirillmakhonin) [inline legion/sdk/legion/sdk/clients/deployment.py]: typo in readiness_pro**beIn**itial_delay, please fix

**Comment by mcdoker18** [comment]
bot, test app

**Review by kirillmakhonin — CHANGES_REQUESTED** [review]

File: legion/toolchains/python/legion/toolchain/metrics.py (hunk 1 above)
Comment (mcdoker18) [reply legion/toolchains/python/legion/toolchain/metrics.py]: Fixed

File: legion/sdk/legion/sdk/clients/deployment.py (hunk 2 above)
Comment (mcdoker18) [reply legion/sdk/legion/sdk/clients/deployment.py]: Fixed

**Comment by mcdoker18** [comment]
bot, test app

**Comment by mcdoker18** [comment]
bot, test app

**Review by kirillmakhonin — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**